tests/Footer.spec.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { shallow } from './enzyme';
+import Footer from '../src/components/Footer/Footer';
+
+describe('Footer test', () => {
+  it('should work fine on Footer', () => {
```

Comment:
Change this to: `it should render without crashing`.

Reply:
Thank you for this feedback. I will revert immediately.